dist/msrcrypto.js

```diff
@@ -8934,9 +8934,11 @@ var workerManager = (function () {
 
             // Send the results to the operation object and it will fire
             //   it's onCompleted event.
-            if (op && e.data.type !== "process") {
+            if (op) {

```

Comment:
Excellent. This passes all the tests. If you copy it to [`dist/msrcrypto.mjs`](https://github.com/kevlened/msrCrypto/blob/master/dist/msrcrypto.mjs#L8937-L8940), I'll 🚢

Reply:
Done!